assets/constants.ts

```diff
@@ -4741,130 +4741,172 @@ export default {
   // stripped out to give a shorter description.
   "SUPPORTED_CONTENT_LANGUAGES": [{
     "code": "en",
-    "description": "English"
+    "description": "English",
+    "dir": "ltr"
```

Comment:
Could we call this "direction"? In general we prefer not to use abbreviations, especially ones that introduce ambiguity (e.g. "dir" could represent "directory").

Reply:
done